torch/optim/_multi_tensor/sgd.py

```diff
@@ -52,8 +53,9 @@ class SGD(Optimizer):
         The Nesterov version is analogously modified.
     """
 
-    def __init__(self, params, lr=required, momentum=0, dampening=0,
-                 weight_decay=0, nesterov=False):
+    def __init__(self, params: _params_t, lr: float = required,
```

Comment:
This passes the tests, but we probably don't want to change this to using a float (I don't know the downstream consequences of this and this looks like a pre-existing problem with the sgd type hints) Would prefer to revert this chance since we're not handling the mypy issues anyways